### PR TITLE
Working V1 Of Kayla Camera

### PR DESCRIPTION
**Pull request by MiladNazeri**
**Test Plan:**

- [x] Run https://hifi-content.s3.amazonaws.com/milad/ROLC/Organize/Projects/Hifi/Scripts/Kayla-Camera/Kayla-Camera.js

- [x] Open up Kayla Camera app

- [x] Edit the name to a new name

- [x] Enable Customer Listener

- [x] Choose Update Position

- [x] Did it Update the position to your current location?

- [x] When you walk around, does the sound seem like it is staying in one place even though you are moving around

- [x] Disable it

- [x] Is your volume back to normal?

- [x] Add a new Position

- [x] Did the new position reflect your current position?

- [x] Edit the name to be new

- [x] Edit the Key to be a certain key press

- [x] Walk away from the area you are currently at

- [x] Press the first Key Press

- [x] Did you teleport to that position?

- [x] Repeat that process for 2-3 different keys and different locations

- [x] Refresh the app in the running list

- [x] When you open the app again, are the settings you had still there?

- [x] Choose Save JSON

- [x] Go to https://kayla-camera.glitch.me/

- [x] Do you see the JSON of your named configuration there?

- [x] Copy the link to the json you just saved

- [x] Delete all the positions you have made and change the config name

- [x] Choose Load JSON URL

- [x] Paste that link into the textbox and choose update

- [x] Do the configurations look the same as you left them previously?

- [x] Do the key presses for the locations work as expected?

- [x] Click the Use Camera Avatar button

- [x] Are you the Camera Avatar?

- [x] Click the use Invisible Avatar Button

- [x] Are you the Invisible Avatar?

- [x] Click Toggle Avatar Collisions

- [x] Are you collisions now opposite of what they were?

- [x] Click Toggle Avatar Collisions again

- [x] Are they the opposite of what they were?

### CODE REVIEW

File: Prototyping/Kayla-Camera/Kayla-Camera.js [hunk 1]
```diff
@@ -0,0 +1,345 @@
...
+            tabletButton.clicked.disconnect(onTabletButtonClicked);
+            tablet.removeButton(tabletButton);
+            tabletButton = null;
+        }
+        tablet = null;
+        Controller.keyPressEvent.disconnect(keyPressHandler);
+    }
+
+    Script.scriptEnding.connect(scriptEnding);
+}());
```
Comment (RebeccaStankus) [inline Prototyping/Kayla-Camera/Kayla-Camera.js]: EOF Line

File: Prototyping/Kayla-Camera/Polyfills.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+                // 7. Return -1.
+                return -1;
+            },
+            configurable: true,
+            writable: true
+        });
+    }
+}
+
+module.exports = registerPolyfills;
```
Comment (RebeccaStankus) [inline Prototyping/Kayla-Camera/Polyfills.js]: EOF line

File: Prototyping/Kayla-Camera/Tablet/Kayla-Camera-Tablet.html [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    <script src="https://code.jquery.com/jquery-3.3.1.min.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/vue/dist/vue.js"></script>
+    <script src="https://cdn.jsdelivr.net/filesaver.js/0.1/FileSaver.min.js"></script>
+    <script src="js/Kayla-Camera-Tablet.js"></script>
+    
+</body>
+
+</html>
```
Comment (RebeccaStankus) [inline Prototyping/Kayla-Camera/Tablet/Kayla-Camera-Tablet.html]: EOF

File: Prototyping/Kayla-Camera/Tablet/css/app.css [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+    -webkit-box-sizing: border-box;
+    -moz-box-sizing: border-box;
+    box-sizing: border-box;
+  }
+  
+  body {
+    font-family: Helvetica Neue, Helvetica, Arial, sans-serif;
+    margin: 0;
+    padding: 0;
+  }
```
Comment (RebeccaStankus) [inline Prototyping/Kayla-Camera/Tablet/css/app.css]: EOF

File: Prototyping/Kayla-Camera/Tablet/js/Kayla-Camera-Tablet.js [hunk 5]
```diff
@@ -0,0 +1,360 @@
...
+            EventBridge.scriptEventReceived.connect(onScriptEventReceived);
+            EventBridge.emitWebEvent(JSON.stringify({
+                type: EVENT_BRIDGE_OPEN_MESSAGE
+            }));
+        }, EVENTBRIDGE_SETUP_DELAY);
+    }
+
+    onLoad();
+
+}());
```
Comment (RebeccaStankus) [inline Prototyping/Kayla-Camera/Tablet/js/Kayla-Camera-Tablet.js]: EOF

File: Prototyping/Kayla-Camera/Tablet/js/Kayla-Camera-Tablet.js [hunk 6]
```diff
@@ -0,0 +1,360 @@
+(function () {
```
Comment (RebeccaStankus) [inline Prototyping/Kayla-Camera/Tablet/js/Kayla-Camera-Tablet.js]: space after function

**Comment by MiladNazeri** [comment]
@RebeccaStankus fixed EOFs and changed the method to handleLookUpString to work with the zone pass

**Pull request merged**